lib/maze/client/appium/bb_client.rb

```diff
@@ -18,6 +18,14 @@ def prepare_session
           end
         end
 
+        def maze_address
+          if Maze.config.aws_public_ip
+            Maze.public_address
+          else
+            "local:#{Maze.config.port}"
+          end
```

Comment:
IIUC there will only be a `Maze.public_address` if there's also a `Maze.config.aws_public_ip`, so this is equivalent:

```suggestion
          Maze.public_address || "local:#{Maze.config.port}"
```

It was a little confusing to me why we were checking `Maze.config.aws_public_ip` only to read `Maze.public_address`

Reply:
Yes, I think that's a fair change to make.  FWIW I think it's still reasonable for both `Maze.config.aws_public_ip` and `Maze.public_address`, as one if set by the user provided options and the other is dynamically determined.